lib/opFns.js

```diff
@@ -581,7 +588,6 @@ module.exports = {
       })
 
       try {
-        checkCallMemCost(runState, options, localOpts)
```

Comment:
Derp. It is never called in that function before. Note to self: shouldn't do these PRs late in the evening.